mms/model_service_worker.py

```diff
@@ -261,7 +237,7 @@ def predict(self, data):
             batch_size = len(req_batch)  # num-inputs gives the batch size
             input_batch, req_id_map, invalid_reqs = self.retrieve_data_for_inference(req_batch, model_service)
             if batch_size == 1:
-                retval.append(model_service.inference([input_batch[0][i] for i in input_batch[0]]))
+                retval.append(model_service.inference(input_batch[0]))
```

Comment:
This should probably be reverted back. How would we pass multiple inputs to a single request?